test/function/samples/emit-file/emit-named-asset-conflict-build-generate/_config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+	skip: true,
```

Comment:
I would actually just delete the tests altogether. Otherwise there is a high danger that the tests rot, and we could restore them from git anyway. If we switch to a warning, the tests need to be rewritten anyway.